packages/internals/utils/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@frctl/utils",
+  "version": "2.0.0-beta.0",
+  "description": "Utilities and helpers for Fractal.",
+  "homepage": "http://fractal.build",
+  "repository": "https://github.com/frctl/fractal/tree/v2/packages/utils",
```

Comment:
Just noticed that the repo location is incorrect - should be `https://github.com/frctl/fractal/tree/v2/packages/internals/utils`